packages/frontend/middleware.ts

```diff
@@ -19,17 +21,16 @@ export async function middleware(request: NextRequest) {
 
   if (ip && !isIPWhitelisted) {
     const redisData = await redis.get(ip)
-    const isIPBlocked = !!redisData
+    const isIPBlocked = redisData === BLOCKED_IP_VALUE
```

Comment:
older values don't make any sense now, similar to refreshing the older blocked ips list

Reply:
Yeah thats fair.  

The only reason for the storage of these was to reduce risk of hitting api limits on ipqs.  

In future may want to have a reset of values at some cadence and store the timestamp when it is saved to handle changes in ip ranges/data centers/vpns etc. Something like this below terrible pseudocode:

``` isIPBlocked redisData[0] === BLOCKED_IP_VALUE && (Date.now() - redisData[1]) / 86400000< 30 ```

then later just setting 2 values, maybe array or something, not super familiar on what possible there, to redis:

``` await redis.set(ip, [BLOCKED_IP_VALUE, Date.now()]) ```

Reply:
yeah agreed thats better. i'll make another PR to implement this.

Reply:
implemented it here: https://github.com/opynfinance/squeeth-monorepo/pull/1015